docs/artifact_repositories/about.md

```diff
@@ -0,0 +1,43 @@
+# Artifact Repositories
+
+Phylum for artifact repositories allows users to configure their artifact
+repositories and/or package managers to block untrusted packages.
```

Comment:
Just a note here:

This is the one place where "package manager" is correct because it's instructing the user to "configure their package manager".

In general, you configure a package manager to use a specific package registry.

This is also the biggest downside of referring to this product as "Phylum for Artifact Repositories". Because these package registries are perfectly usable even for an individual or company that does not use an artifact repository.